app/code/Magento/CustomerImportExport/Model/Import/Customer.php

```diff
@@ -270,13 +270,23 @@ protected function _saveCustomerEntities(array $entitiesToCreate, array $entitie
             $this->_connection->insertOnDuplicate(
                 $this->_entityTable,
                 $entitiesToUpdate,
-                $this->customerFields
+                $this->getCustomerEntityFieldsToUpdate($entitiesToUpdate)
             );
         }
 
         return $this;
     }
 
+    private function getCustomerEntityFieldsToUpdate(array $entitiesToUpdate): array
```

Comment:
Could you maybe a docblock to this method with a short explanation of the bug this method fixes. Is there actually a github issue for this fix cause you could add a link to that here.